test/models/prx_importer_test.rb

```diff
@@ -235,15 +247,23 @@ def self.new_guid
       doc = prx_importer.import(prx_story_id: 87683)
 
       doc.title.must_equal "Virginity, Fidelity, and Fertility"
-      doc.tags.sort.must_equal  ["PRX", "Weekly Program"]
+      doc.tags.sort.must_equal  ["Weekly Program"]
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.